src/codegen/mod.rs

```diff
@@ -1487,6 +1500,17 @@ impl ToRustTy for Type {
                     }
                 }
             }
+            TypeKind::Complex(fk) => {
+                use ir::ty::FloatKind;
+                let float_path = match fk {
```

Comment:
Note that there's a new option to use `c_float`/`c_double`. You should probably respect that here, reusing the same code.